src/model/frontmatter.ml

```diff
@@ -71,5 +85,13 @@ let parse_short_title loc t =
         (Error.make
            "Short titles cannot contain other block than a single paragraph" loc)
 
+let parse_toc_status loc (t : tag_payload) =
+  match t with
+  | [
+   { Location_.value = `Paragraph [ { Location_.value = `Word "open"; _ } ]; _ };
+  ] ->
+      Result.Ok (Location_.at loc (Toc_status `Open))
+  | _ -> Error (Error.make "@toc_status can only take the 'open' value" loc)
```

Comment:
Do you have any other values in mind that this might take?

Reply:
The value I had in mind was `hidden` or `private` where it is both open and not rendered as a link.